plugins/rikki/heroeslounge/http/Match.php

```diff
@@ -78,25 +83,40 @@ public function games($id)
         return $retVal;
     }
 
-    public function getTodaysMatches($date, $tz1 = TimezoneHelper::DEFAULT_TIMEZONE, $tz2 = "")
+    private function formatTimezone($tz1, $tz2)
     {
         if ($tz2) {
-            $timezone = $tz1 . '/' . $tz2;
+            return $tz1 . '/' . $tz2;
         } else {
-            $timezone = $tz1;
+            return $tz1;
         }
+    }
 
-        // get today in a given timezone
-        if ($date == 'today') {
-            $start = Carbon::today($timezone);
+    private function getMatchesForDate($start, $isPlayed)
```

Comment:
I like getMatchesForDay better as name - this is also very close to getMatchesByDate

Reply:
I would call this `getDbDayMatches`